package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "dev": "nodemon --config nodemon.config.json -x 'pnpm build:packages'",
     "build": "turbo run build --cache-dir=.turbo",
-    "build:packages": "turbo run build --filter=!docs --filter=!template-*",
+    "build:packages": "turbo run build --filter=!docs --filter=!template-* --filter=!demo-*",
```

Comment:
Demos were made to serve as test/validators.

Reply:
> Demos were made to serve as test/validators.

@arboleya I suppose I was thinking that the `build:packages` command should only be responsible for building the packages.

Then the `build` command would be `build` + `validate` as you say.

Thoughts?